src/clipboard_handlers/data_validation_clipboard.ts

```diff
@@ -143,18 +138,18 @@ export class DataValidationClipboardHandler extends AbstractCellClipboardHandler
   private adaptDataValidationRule(
     sheetId: UID,
     rule: DataValidationRule,
-    toAdd: string[],
-    toRemove: string[]
+    toAdd: Zone[],
+    toRemove: Zone[]
   ) {
-    const dvRangesXcs = rule.ranges.map((range) => this.getters.getRangeString(range, sheetId));
-    const newRangesXC = recomputeZones([...dvRangesXcs, ...toAdd], toRemove);
-    if (newRangesXC.length === 0) {
+    const dvRangesXcs = rule.ranges.map((range) => range.zone);
+    const newRangesZones = recomputeZones([...dvRangesXcs, ...toAdd], toRemove);
```

Comment:
```suggestion
    const dvZones= rule.ranges.map((range) => range.zone);
    const newDvZones = recomputeZones([...dvRangesXcs, ...toAdd], toRemove);
```
or something like that